host/bloodview/src/device.c

```diff
@@ -557,13 +557,8 @@ static bool device__queue_msg_led(bool enable_lights)
 	return true;
 }
 
-/**
- * Get the appropriate source for a given channel, according to acq mode.
- *
- * \param[in]  channel  The channel to find the source for.
- * \return source for the channel.
- */
-static enum bl_acq_source device__get_channel_source(uint8_t channel)
+/* Exported function, documented in device.h */
+enum bl_acq_source device__get_channel_source(uint8_t channel)
```

Comment:
Double underscore is for static functions!

```suggestion
enum bl_acq_source device_get_channel_source(uint8_t channel)
```

Reply:
+1